### PR TITLE
Adds Ketrazine, a powerful combat stimulant with severe side-effects which can be synthesized by the crew

### PR DESCRIPTION
**Pull request by Ordonis**
:cl: Ordo
rscadd: Adds Ketrazine, a powerful but dangerous combat stim which the crew can synthesize
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1105,3 +1105,107 @@
...
+			M.adjustKnockdown(70*REM,0)
+			M.drop_all_held_items()
+			M.Dizzy(5)
+			M.drowsyness +=6
+			M.adjustBruteLoss(20*REM,0)
+			M.adjustToxLoss(15*REM,0)
+			M.adjustStaminaLoss(40*REM,0)
+			M.losebreath +=1
+
+			=1
```
Comment (coiax) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: =1?

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 2]
```diff
@@ -226,3 +226,18 @@
 	id = "corazone"
 	results = list("corazone" = 3)
 	required_reagents = list("phenol" = 2, "lithium" = 1)
+
+/datum/chemical_reaction/morphine
+	name = "Morphine"
+	id = "morphine"
+	results = list("morphine" = 3)
+	required_reagents = list("carbon" = 2, "hydrogen" = 2, "nitrogen" = 1, "oxygen" = 1)
+	required_temp = 480
```
Comment (coiax) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: Did you mean to include this?

Comment (Ordonis) [reply code/modules/reagents/chemistry/recipes/medicine.dm]: No, I am a double dumbfuck for first pushing this entire thing to the morphine PR, and now for leaving the morphine thing in this, apologies

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (Ordonis) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Is that not required at the end of a (switch)? I was under the impression that it was.

**Comment by GuyonBroadway** [comment]
Isn't this basically just super bathsalts?

**Comment by GuppyLaxx** [comment]
1. Horribly spammed because le meme stimulant
2. Barely used, if ever, due to how crippling it can be
3. Ends up more used as a toxin due to the fact it kills you in a tiny amount of time if ODd, if somehow  removed in time there's still a horrible addiction

**Comment by XDTM** [comment]
You can eat 4u pills of this to get near-adminordrazine healing and stun removal for 16 seconds safely. Then again, it's hard to make and temporary so it's probably fine.

**Comment by Ordonis** [comment]
The idea was to have a more consequence-filled version of the combat stims that syndies get and/or have a 'are you sure you want to go loud/robust?' chem. I'm more than happy to balance it in whatever direction people feel it needs balancing, but it always struck me as odd that for war ops or other confirmed threats there wasn't a chemical that 
a) rewarded chemistry for making a variety of useful medicines and not using them all to make le ebin druggs
b) was a very definite window of opportunity chem: you will NOT come off better than when you started with this, but if you need to charge headlong into bleb or other known threats, might keep you alive long enough to do something useful
c) rewards medbay for being near the frontlines with a variety of chems; combat medic always feels a little 'shove tricord and meth into them and damn the consequences, then watch as they get slaughtered' rather than 'use this stimulant and immediately prepare for the consequences that will come afterwards, having your less deadly healing chems on hand
d) punishes foolish toxmemers who think 'oh I'll just poison them with this' only to get robusted in the 15 seconds before hell begins to break loose
e) makes use of chemicals that don't necessarily get made/kept for their own purposes that much.

**Comment by ma44** [comment]
Inb4 odysseus syringe gun comes along. 

Tho it would probably be more far if the damage from part 1 of the addition would be moved up, as it would give the player the chance to seek out the drug instead of just immediately taking damage.

**Comment by Ordonis** [comment]
So remove the brute and oxy damage from the first stage entirely?

**Comment by ma44** [comment]
You can make the later tiers probably deal more damage, as the player is given a warning.

**Comment by Tlaltecuhtli** [comment]
maybe give it some ingredient that cant be made in chen like gold or earthblood seems ez to make

**Comment by Ordonis** [comment]
I'd rather add in something like gold as a catalyst rather than a full-on ingredient; cooperation is easier when you only need to give tiny amounts. I think I might go with gold or similar, simply because it turns the 'gib H2SO4' into 'trade H2SO4 for Au'.

**Comment by Tlaltecuhtli** [comment]
barter gold apples for mutagen

**Comment by Ordonis** [comment]
I am absolutely certain the response to that proposed trade would be greentiding culminating with either the death of all botanists, the firebombing of botany after a dispenser was stolen, the firebombing of chemistry after a chemist makes a mistake in building a firebomb, the death of all chemists, or the delamination of the supermatter. 

R&D can't kill you too easily until later in the round, when they should have gold to spare.

**Comment by XDTM** [comment]
Thing is, you _will_ come off better than you started as long as you use small doses, then rinse and repeat. Smoking this with a vape will be very powerful, as long as you let the chem wear out to reset the current_stage every once in a while.

**Comment by Ordonis** [comment]
What's the reagent distro of a vape pen? I could add in a really slow but constant brainDamage as a way to discourage constant low intensity use if you think that would improve balance

**Comment by XDTM** [comment]
I'm not sure what would be an ideal solution, maybe make it so it won't heal if you don't have at least 5 units. Addiction shouldn't be necessary if you do that.

**Comment by WJohn** [comment]
This is fairly cool, I like a high risk, high reward kind of chem. Good stuff 👍

**Comment by Shapsy** [comment]
This just sounds like it'd be a nigh-instawin if the user uses it right, and instant death if they fuck it up, which doesn't really sound fun

**Comment by yorii** [comment]
Why not use a real chemical name? Like Modafinil for example, which is a chemical actually used by military forces worldwide today much like amphetamine was used during ww2.

**Comment by Ordonis** [comment]
I've been told in the past to use non-realworld names for things; I was honestly thinking of calling it Pervitin (considering how nasty that ended up being) but that would skate the NSDAP rule, and I wasn't sure if Modafinil would run afoul of IP law. I'd much prefer to have a realworld or realworldish name, as opposed to NOT-KETRACEL.

**Comment by Supermichael777** [comment]
We need to get over the lethal ODod concept it leads to unbalanced chems that are more useful as the supertoxin of the week

Its okay to have a chem that deals constant damage(producing a second chem can also work for a lingering effect)

**Comment by AdamElTablawy** [comment]
This is pretty much a deathmix chem. The guy is strong for a little bit and then fucking dies very rapidly. 

Also fairly easy to synthesize, because m a c r o

**Comment by Ordonis** [comment]
You do need gold to synthesize it as well as heat, so macromemes will only get you partway there. Is there a general consensus that the downside needs to be less 'u dun goofd'? Because it doesn't have instakills, it is survivable if you've taken precautions ahead of time to heal yourself when the hulk-esque effect ends. It's also a means of allowing a very, very short window of opportunity for medical to help the crew fend off serious threats; it won't stop you from dying if a syndie is laying down massive amounts of fire, but you might soak a few more bullets or get close enough to interfere. Same goes with cultish threats. At the same time, the eternally pox-ridden greytide will find that they probably can't achieve much mischief with it if they manage to get their hands on it. The CMO can already make nastier deathmixes with chloral and friends.

**Comment by yorii** [comment]
Understandable, I also realized this is nothing much at all like modafinil, I'm going to PR my own modafinil. Modafinil is also not a brand name but the actual chemical name under no actual IP.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 3]
```diff
@@ -1105,3 +1105,101 @@
...
+	M.adjustBruteLoss(-5*REM, 0)
+	M.adjustFireLoss(-5*REM, 0)
+	M.adjustOxyLoss(-5*REM, 0)
+	M.AdjustStun(-80*REM, 0)
+	M.AdjustKnockdown(-70*REM, 0)
+	M.adjustStaminaLoss(-80*REM, 0)
+	M.AdjustUnconscious(-50*REM, 0)
+	M.adjustBrainLoss(0.5*REM,0)
+	switch(current_cycle)
+		if(13)
```
Comment (DaedalusGame) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: should be 0 to 13

Comment (DaedalusGame) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: should be lower than 13 probably, or you'll get this message and then next tick your muscles begin to ache terribly

Comment (Ordonis) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Query: if the if(n) is set to something like 1 or 2, does that mean the positive effects will stop? That was my reason for leaving the message so late.

Comment (DaedalusGame) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: no, the positive effects never stop because you didn't write it that way.

Comment (Ordonis) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: Can I just add in a (low) probability check to the messages so that they appear occasionally during each stage?

Comment (DaedalusGame) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: yepp, just do if(prob(20)) or smth

**Comment by AdamElTablawy** [comment]
@ordonis

Three things.

1. Chloral is worthless as a deathmix chem. Never speak of it. It is forbidden.

2. Truthfully, the downsides are so significant that short of occasionally briefly popping it it’s functionally useless, because given the time and resource investment you could have made a mix with all the upsides and none of the downsides in the same timeframe, without having to worry about dying if you fuck it up.

3. As a deathmix chem this is actually very good. You can make mixes that kill faster, but considering this gives the illusion of being healthy and healing you, it likely won’t attract attention or attacks. Sure a syringe gun will, but using it as a poison or in smoke will likely incite nobody, until it’s too late. Look at how effective amantin is-it has no upsides but because it seems harmless and is mostly invisible people don’t respond even to being hyposprayed with the stuff. 

Tl;dr it’s not overpowered at all, but it actually serves better as a death chem than a combat stim since you can make better combat mixes in less time with less effort and less side effects.
